docs/src/main/sphinx/connector/hive.rst

```diff
@@ -408,6 +428,8 @@ Property Name                                      Description
                                                    managed tables.
                                                    See the :ref:`hive_table_properties` for more information
                                                    on auto_purge.
+
+``hive.partition-projection-enabled``              Enables Athena partition projection support                     ``false``
```

Comment:
If it is false by default .. why do we talk about disabling it if there are issues in the earlier section instead of needing to turn it on?